packages/framework/src/Facades/Vite.php

```diff
@@ -5,37 +5,70 @@
 namespace Hyde\Facades;
 
 use Illuminate\Support\HtmlString;
+use InvalidArgumentException;
 
 /**
  * Vite facade for handling Vite-related operations.
  */
 class Vite
 {
+    protected const CSS_EXTENSIONS = ['css', 'less', 'sass', 'scss', 'styl', 'stylus', 'pcss', 'postcss'];
+    protected const JS_EXTENSIONS = ['js', 'jsx', 'ts', 'tsx'];
```

Comment:
This is seriously overkill, but it doesn't really add any more complexity to add all these. While they might not work unless more media file extensions are registered, adding all these here future proofs things without needing to support customization here.